webtau-docs/webtau/reference/table-data.md

```diff
@@ -29,24 +29,36 @@ Use `permute(v1, v2)` to automatically generate multiple rows.
 
 ```tabs
 Groovy:
-:include-groovy: com/twosigma/webtau/data/table/TableDataExtensionTest.groovy {entry: "createTableDataWithPermute", bodyOnly: true}
+:include-groovy: com/twosigma/webtau/data/table/TableDataGroovyTest.groovy {entry: "createTableDataWithPermute", bodyOnly: true}
 
 Java:
-:include-groovy: com/twosigma/webtau/data/table/TableDataTest.groovy {entry: "createTableDataWithPermute", bodyOnly: true}
+:include-java: com/twosigma/webtau/data/table/TableDataJavaTest.java {entry: "createTableDataWithPermute", bodyOnly: true, removeReturn: true, removeSemicolon: true}
 ```
 
 :include-table: table-with-permute.json
 
-# Previous Value Reference
+# Cell Above Value Reference
 
-Use `cell.previous` to refer to the previous row value
+Use `cell.above` to refer to the previous row value
 
 ```tabs
 Groovy:
-:include-groovy: com/twosigma/webtau/data/table/TableDataExtensionTest.groovy {entry: "createTableDataWithPreviousRef", bodyOnly: true}
+:include-groovy: com/twosigma/webtau/data/table/TableDataGroovyTest.groovy {entry: "createTableDataWithAboveRef", bodyOnly: true}
 
 Java:
-:include-groovy: com/twosigma/webtau/data/table/TableDataTest.groovy {entry: "createTableDataWithPreviousRef", bodyOnly: true}
+:include-java: com/twosigma/webtau/data/table/TableDataJavaTest.java {entry: "createTableDataWithAboveRef", bodyOnly: true, removeReturn: true, removeSemicolon: true}
 ```
 
-:include-table: table-with-cell-above.json
\ No newline at end of file
+:include-table: table-with-cell-above.json
+
+You can use underscores of various length as a shortcut for `cell.above`
+
+```tabs
+Groovy:
+:include-groovy: com/twosigma/webtau/data/table/TableDataGroovyTest.groovy {entry: "createTableDataWithAboveRefShortcut", bodyOnly: true}
+
+ Note: Unlike Java, Groovy can use underscores of any length as long as it is longer than 1 character.
```

Comment:
I'd suggest a slight rewording:
In Groovy, unlike Java, you can use underscores of any length for the shortcut as long as ...

Reply:
Thank you